holoviews/core/io.py

```diff
@@ -854,3 +854,10 @@ def contents(self, maxlen=70):
     def listing(self):
         "Return a list of filename entries currently in the archive"
         return ['.'.join([f,ext]) if ext else f for (f,ext) in self._files.keys()]
+
+    def clear(self):
+        "Clears the file archive"
+        self._files.clear()
```

Comment:
You seem to be basing your PRs off an commit, which keeps reintroducing these changes, which makes it harder to review your PRs.